frontends/web/src/routes/lightning/receive/receive.tsx

```diff
@@ -233,7 +233,7 @@ export function Receive() {
         <View fitContent textCenter verticallyCentered>
           <ViewContent withIcon="success">
             <p>{t('lightning.receive.success.message')}</p>
-            <span>{inputSatsText} sats (<FiatConversion amount={invoiceAmount} noBtcZeroes />)</span>
+            <span>{} sats (<FiatConversion amount={invoiceAmount} noBtcZeroes />)</span>
```

Comment:
hm I think you prob want to keep the `inputSatsText` here, also should this FiatConversion have alwaysShowAmounts prop?